tests/testsuite/doc.rs

```diff
@@ -1421,3 +1421,33 @@ fn doc_cap_lints() {
     );
 
 }
+
+#[test]
+fn doc_message_format() {
+    if !is_nightly() {
+        // This can be removed once 1.30 is stable (rustdoc --error-format stabilized).
+        return;
+    }
+    let p = project().file("src/lib.rs", "asdf").build();
```

Comment:
Hm, let's use a code that is valid, but has docstring errors? That way, we'll be more sure that we are checking rustdoc output specifically.

Reply:
Sure, I was on the fence with that.  My concern is that the rustdoc messages might be a little unstable, but there is another test already doing that.